website/source/guides/operations/requirements.html.md

```diff
@@ -67,8 +67,9 @@ port.
 * HTTP API (Default 4646). This is used by clients and servers to serve the HTTP
   API. TCP only.
 
-* RPC (Default 4647). This is used by servers and clients to communicate among
-  each other. TCP only.
+* RPC (Default 4647). This is used for internal RPC communication between agents
```

Comment:
The RPC layer is used for consensus and for RPC forwarding, so just calling out the consensus algorithm is slightly wrong. I suggest it changes to: `between client agents and servers, and for inter-server traffic.`